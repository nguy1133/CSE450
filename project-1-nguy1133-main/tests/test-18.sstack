// A bunch of PUSHes and POPs (from example in project 1 description).
PUSH "FIRST"
PUSH "next"
POP var1
PUSH 'March'
PUSH '4th'
POP day
POP month
PUSH day
PRINT var1 " " month " " day
