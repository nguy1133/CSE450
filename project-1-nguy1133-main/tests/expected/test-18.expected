next March 4th
